urdf_model/include/urdf_model/color.h

```diff
@@ -73,13 +73,13 @@ class Color
       {
         try
         {
-          rgba.push_back(std::stof(pieces[i]));
+          double piece = strToDouble(pieces[i].c_str());
+          if ((piece < 0) || (piece > 1))
+            throw ParseError("Component [" + pieces[i] + "] is outside the valid range for colors [0, 1]");
+          rgba.push_back(piece);
```

Comment:
Looks like this line warns on windows

```
c:\j\workspace\ci_packaging_windows\ws\install\include\urdf_model\color.h(79): warning C4244: 'argument': conversion from 'double' to 'const float', possible loss of data [C:\J\workspace\ci_packaging_windows\ws\build\robot_state_publisher\robot_state_publisher_solver.vcxproj
```

https://ci.ros2.org/job/ci_packaging_windows/82

Reply:
Ug.  Well, I guess we can `static_cast<float>` here; we know it will be safe.

Reply:
#48